tests/model_explainability/evalhub/test_evalhub_networkpolicy.py

```
@@ -0,0 +1,265 @@
+            match_labels = (spec.podSelector or {}).get("matchLabels", {})
+            if not all(match_labels.get(k) == v for k, v in evalhub_labels.items()):
+                continue
+
+            if not _policy_restricts_metrics_port(spec):
+                continue
+
+            # Policy restricts the metrics port — every restricting rule must allow monitoring.
+            # An empty ingress list means deny-all, which never allows monitoring.
+            ingress_rules = spec.get("ingress") or []
+            if not ingress_rules:
+                assert False, (
```

Ruff (0.15.12)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)